src/consumer.ts

```diff
@@ -198,7 +199,7 @@ export class Consumer extends TypedEventEmitter {
     let currentPollingTimeout = this.pollingWaitTimeMs;
     this.receiveMessage({
       QueueUrl: this.queueUrl,
-      AttributeNames: this.attributeNames,
+      AttributeNames: this.attributeNames as QueueAttributeName[],
```

Comment:
What's the reason for this, is there a new TS error here? One that I might have to look into as there might be a better way of typing this.

Reply:
Yes, there's a new typing added for that field, so the existing this.attributeNames (which is string[]  typed) was raising an error there.

I wasn't sure if we should change the typing for this.attributeNames or not to match; I leave this decision up to you.

Reply:
I've pushed a commit up to change or types to respect this change.